plugins/ohifv3/extensions/monai-label/src/tools/ProbeMONAILabelTool.ts

```diff
@@ -1,4 +1,16 @@
-import { Types, metaData, utilities as csUtils } from '@cornerstonejs/core';
+/*
+Copyright (c) MONAI Consortium
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
It's your repository and your company, but a license at the root of the repository might be sufficient and legally binding for all sub-files, no?

Reply:
just following the previous practice/guidelines.